boudicca.base/eventcollector-client/src/main/kotlin/base/boudicca/api/eventcollector/EventCollectorDebugger.kt

```diff
@@ -24,19 +23,14 @@ class EventCollectorDebugger {
         collectedEvents.forEach {
             println(it)
         }
-        println(Collections.getAllPastCollections()[0])
+        val fullCollection = Collections.getAllPastCollections()[0]
```

Comment:
is this basically equivalent to Collections.getLastCollection() ?

Reply:
kinda, but it would make sense to add a helper method for this, i will do this